src/service.js

```diff
@@ -83,20 +88,27 @@ LS2Request.prototype.send = function() {
 		}
 		if(!self.subscribe) {
 			self.cancel();
+		} else if(self.ts && refs[self.ts]) {
+			delete refs[self.ts];
 		}
 	};
+	self.ts = performance.now();
```

Comment:
I'm not a big fan of that idea, as in the case of multiple calls to the the same service with identical parameters could exist validly, and could potentially result in old subscripts or pending requests to be GC'd and the javascript side never executing. Having a completely unique id guarantees accidental GC will be avoided.